maps/~map_system/maps.dm

```diff
@@ -194,14 +194,9 @@ var/list/all_maps = list()
 // Get a list of 'nearby' or 'connected' zlevels.
 // You should at least return a list with the given z if nothing else.
 /datum/map/proc/get_map_levels(var/srcz, var/long_range = FALSE, var/om_range = -1)
-	//Overmap behavior
-	if(use_overmap)
```

Comment:
Slightly concerned that this is going to have knock-on effects for servers that use the overmap but have set up their announcements to work with the current logic (via a shared overmap sector or something like that).

Reply:
Thinking about it, why not just move the announcement computer onto the player z-levels?

Reply:
This does not change any announcements already coming from an overmap sector. It just allows for those not on overmap sectors to fall back on the old behavior.

Reply:
Yes, which isn't really 1:1 with previous code - so you may end up with offmap Z-levels now announcing on the main levels.
It's a niche issue and not one that applies to Polaris but I figured it was worth raising.

Reply:
That sounds like a fairly esoteric issue. Think this is fine.